lib/rules/key-spacing.js

```diff
@@ -291,13 +318,15 @@ module.exports = function(context) {
     /**
      * Verifies spacing of property conforms to specified options.
      * @param  {ASTNode} node Property node being evaluated.
+     * @param {boolean} singleLine True if single line check
      * @returns {void}
      */
-    function verifySpacing(node) {
-        var whitespace = getPropertyWhitespace(node);
-        if (whitespace) { // Object literal getters/setters lack colons
-            report(node, "key", whitespace.beforeColon, beforeColon);
-            report(node, "value", whitespace.afterColon, afterColon);
+    function verifySpacing(node, singleLine) {
```

Comment:
Same as for `report`, rather than a boolean `singleLine` flag, can you pass in the expected options instead?